### PR TITLE
fix(frontend): the default scrollbar is present on the "Add course" screen with the custom scrollbar tm-651

### PR DESCRIPTION
**Pull request by Vfivew**
modalisopen

### CODE REVIEW

File: apps/frontend/src/libs/components/modal/modal.tsx [hunk 1]
```diff
@@ -40,6 +41,8 @@ const Modal: React.FC<Properties> = ({
 		isOpen && styles["active"],
 	);
 
+	useToggleScroll(isOpen);
```
Comment (what1s1ove) [inline apps/frontend/src/libs/components/modal/modal.tsx:44]: to it just in `useEffect`, w/o a custom hook

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/hooks/use-toggle-scroll/use-toggle-scroll.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { useEffect } from "~/libs/hooks/hooks.js";
+
+function useToggleScroll(isOpen: boolean): void {
```
Comment (what1s1ove) [inline apps/frontend/src/libs/hooks/use-toggle-scroll/use-toggle-scroll.ts:3]: an arrow function

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/hooks/use-toggle-scroll/use-toggle-scroll.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { useEffect } from "~/libs/hooks/hooks.js";
+
+const useToggleScroll = (isOpen: boolean): void => {
+	useEffect(() => {
+		if (isOpen) {
+			document.body.classList.add("no-overflow");
+		} else {
+			document.body.classList.remove("no-overflow");
+		}
```
Comment (what1s1ove) [inline apps/frontend/src/libs/hooks/use-toggle-scroll/use-toggle-scroll.ts:9]:
```suggestion
			document.body.classList.toggle("no-overflow", isOpen);
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/hooks/hooks.ts [hunk 4]
```diff
@@ -5,6 +5,7 @@ export { useAppTitle } from "./use-app-titile/use-app-title.hooks.js";
 export { useHandleClickOutside } from "./use-handle-click-outside/use-handle-click-outside.hook.js";
 export { useHandleEscPress } from "./use-handle-esc-press/use-handle-esc-press.hook.js";
 export { usePagination } from "./use-pagination/use-pagination.hook.js";
+export { useToggleScroll } from "./use-toggle-scroll/use-toggle-scroll.js";
```
Comment (GEOFARL) [inline apps/frontend/src/libs/hooks/hooks.ts:8]:
```suggestion
export { useToggleScroll } from "./use-toggle-scroll/use-toggle-scroll.hook.js";
```

**Review by Oleksandra-Masiuk — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Comment by what1s1ove** [comment]
@Vfivew resolve conflicts

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**